app/views/exams/ProgressPage.vue

```diff
@@ -27,7 +27,7 @@
         type="button"
         class="btn btn-lg btn-success"
         value="Mark as Complete"
-        @click="submit"
+        @click="() => submit(false)"
```

Comment:
why ? i think "submit" is same as "() => submit(false)"

Reply:
no, it sends PointerEvent when it's empty. JS thinks of it as true because we don't actually check `true` boolean value